deploy/config/rarimo-dev.ts

```diff
@@ -0,0 +1,9 @@
+export const treeSize = 80;
+export const evidenceRegistry = "0x621415816395372D40567B950B98BF0146b7DBdC";
```

Comment:
Why is it different?

Reply:
It is temp address for testing only (manual deployment)